prime_numbers.py

```diff
@@ -0,0 +1,31 @@
+def generate_primes(n=100):
+    """
+    This function generates all prime numbers between 1 to n.
+    A prime number is a natural number greater than 1 that has no positive divisors other than 1 and itself.
+    The numbers 0 and 1 are not prime numbers.
+
```

Comment:
The prime number generation function can be optimized by checking divisibility only up to the square root of the possiblePrime. This will reduce the complexity of the function.